official/vision/data/create_coco_tf_record.py

```diff
@@ -210,27 +210,30 @@ def bbox_annotations_to_feature_dict(
   data, num_skipped = coco_annotations_to_lists(
       bbox_annotations, id_to_name_map, image_height, image_width,
       include_masks)
-  feature_dict = {
-      'image/object/bbox/xmin':
-          tfrecord_lib.convert_to_feature(data['xmin']),
-      'image/object/bbox/xmax':
-          tfrecord_lib.convert_to_feature(data['xmax']),
-      'image/object/bbox/ymin':
-          tfrecord_lib.convert_to_feature(data['ymin']),
-      'image/object/bbox/ymax':
-          tfrecord_lib.convert_to_feature(data['ymax']),
-      'image/object/class/text':
-          tfrecord_lib.convert_to_feature(data['category_names']),
-      'image/object/class/label':
-          tfrecord_lib.convert_to_feature(data['category_id']),
-      'image/object/is_crowd':
-          tfrecord_lib.convert_to_feature(data['is_crowd']),
-      'image/object/area':
-          tfrecord_lib.convert_to_feature(data['area'], 'float_list')
-  }
-  if include_masks:
-    feature_dict['image/object/mask'] = (
-        tfrecord_lib.convert_to_feature(data['encoded_mask_png']))
+  feature_dict = {}
+    
+  if not (len(bbox_annotations) == num_skipped):
+      feature_dict = {
```

Comment:
The indentation here seems not consistent with the rest.

Reply:
I have adjusted the indentation @luotigerlsx, please check once